<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pert Algorithme Modeling</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/icon/MoPert.ico</normaloff>:/new/prefix1/icon/MoPert.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>10</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="pertView">
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="openButton">
            <property name="text">
             <string>Open</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/open.ico</normaloff>:/new/prefix1/icon/open.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="repeatButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/repeat.ico</normaloff>:/new/prefix1/icon/repeat.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="previousButton">
            <property name="text">
             <string>previous</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/rewind.ico</normaloff>:/new/prefix1/icon/rewind.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="playButton">
            <property name="text">
             <string>play</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/play.ico</normaloff>:/new/prefix1/icon/play.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="stopButton">
            <property name="text">
             <string>stop</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/stop.ico</normaloff>:/new/prefix1/icon/stop.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="nextButton">
            <property name="text">
             <string>next</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/new/prefix1/icon/forward.ico</normaloff>:/new/prefix1/icon/forward.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Timer :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timerSpinBox">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="stepLabel">
            <property name="text">
             <string>Step 1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="scaleSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>110</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="outputText">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>sceneScaleChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>playButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repeatButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>repeatButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>previousButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timerSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>timerChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openButtonClicked()</slot>
  <slot>sceneScaleChanged(int)</slot>
  <slot>playButtonClicked()</slot>
  <slot>stopButtonClicked()</slot>
  <slot>repeatButtonClicked()</slot>
  <slot>previousButtonClicked()</slot>
  <slot>nextButtonClicked()</slot>
  <slot>timerChange(int)</slot>
 </slots>
</ui>
